lib/course_planner/accounts/user.ex

```diff
@@ -46,6 +46,12 @@ defmodule CoursePlanner.Accounts.User do
       greater_than_or_equal_to: 1, less_than_or_equal_to: 7)
   end
 
+  def changeset(model, params, :bulk) do
+    model
+    |> changeset(params)
+    |> validate_required([:role])
```

Comment:
I think we should add this validation to the default changeset and remove the bulk changeset, it doesn't make sense in general to have a user without role, they won't have permission for anything

Reply:
well, I think we should do it in another PR. 12 test from other controllers will break

Reply:
I opened a pr for adding role to user changeset "https://github.com/digitalnatives/course_planner/pull/225"

Reply:
I merged it, you may use that already

Reply:
done